src/uu/pmap/src/smaps_format_parser.rs

```diff
@@ -0,0 +1,461 @@
+// This file is part of the uutils procps package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+use crate::maps_format_parser::{parse_map_line, MapLine};
+use std::io::{Error, ErrorKind};
+
+// Represents a parsed single entry from /proc/<PID>/smaps for the extended formats.
+#[derive(Debug, Clone, Default, PartialEq)]
+pub struct SmapEntry {
```

Comment:
What's the reason for not representing the fields `Pss_Dirty` and `KSM` from `/proc/<PID>/smaps`?